rotkehlchen/chain/ethereum/modules/l2/loopring.py

```diff
@@ -322,7 +322,7 @@ def get_account_balances(self, account_id: int) -> Dict[Asset, Balance]:
         for balance_entry in response:
             try:
                 token_id = balance_entry['tokenId']
```

Comment:
so is `token_id` always returned as an int by the json? Are you 100% sure? If not then do the same for this.

Reply:
Yes I'm sure. I read https://docs3.loopring.io/en/dex_apis/getTokens.html and the expected type is integer. Also I queried the list and double checked that the information is correct.